draft-ietf-acme-acme.md

```diff
@@ -1950,16 +1950,16 @@ client then provisions the key authorization as a resource on the HTTP server
 for the domain in question.
 
 The path at which the resource is provisioned is comprised of the fixed prefix
-".well-known/acme-challenge/", followed by the "token" value in the challenge.
-The value of the resource MUST be the ASCII representation of the key
-authorization.
+".well-known/acme-challenge/", followed by the SHA-256 digest of the "token"
+value in the challenge. The value of the resource MUST be the ASCII
```

Comment:
I think base64url encoding makes perfect sense.

A hash of the token (instead of the entire key authorization) was proposed for consistency with SAN A from `tls-sni-02`. This would result in a de facto pattern that the SHA256 of the token is always used in requests, which could be applied to future challenge types.